packages/bruno-app/src/components/Sidebar/Collections/Collection/CollectionItem/GenerateCodeItem/utils/snippet-generator.js

```
@@ -1,51 +1,71 @@
-      const authHeaders = getAuthHeaders(collectionAuth, request.auth);
-      headers = [...headers, ...authHeaders];
+      const collectionAuth = collection?.draft?.root ? get(collection, 'draft.root.request.auth') : get(collection, 'root.request.auth');
+      headers = [...headers, ...getAuthHeaders(collectionAuth, request.auth)];
     }
 
-    // Interpolate headers and body if needed
     if (shouldInterpolate) {
       headers = interpolateHeaders(headers, variables);
       if (request.body) {
-        request.body = interpolateBody(request.body, variables);
+        request = { ...request, body: interpolateBody(request.body, variables) };
```

Biome (2.1.2)

[error] Can't assign request because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)